### ISSUE TITLE
rultor is replying to an irrelevant user

### ISSUE THREAD

**Issue by adtac** [yegor256/rultor#1205]
https://github.com/coala/coala/pull/4637

rultor is replying to an user called `@jeff` - he is not a part of our org and he certainly didn't call a `@rultor merge`. User `@gureks` called rultor but then deleted his comment.

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by 0crat** [comment]
Job `gh:yegor256/rultor#1205` assigned to @yegor256, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by yegor256** [comment]
@rultor release, tag is `1.66.2`

**Comment by yegor256** [comment]
@adtac thanks for reporting that, definitely a bug on our side. It's fixed in e7bb099 Please close the ticket if it's OK.

**Comment by 0crat** [comment]
I can't do it for technical reasons, I'm very sorry. If you don't know what to do, submit this error as a ticket [here](https://github.com/zerocracy/datum):

```
1.0-SNAPSHOT 4f767b8 2017-10-12T18:11:51
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/scope/wbs/remove_job_from_wbs.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/scope/wbs/remove_job_from_wbs.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:86)
	at com.zerocracy.farm.reactive.brigade.StkWithStatus.process(StkWithStatus.java:55)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:101)
	at com.zerocracy.farm.reactive.Brigade.lambda$process$0(Brigade.java:83)
	at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:66)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.value(LengthOf.java:56)
	at org.cactoos.iterable.LengthOf.value(LengthOf.java:56)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:85)
	at com.zerocracy.farm.reactive.Flush.process(Flush.java:86)
	at com.zerocracy.farm.reactive.Flush.exec(Flush.java:71)
	at com.zerocracy.fa...
```
